teps/0128-scheduled-runs.md

```diff
@@ -63,7 +64,100 @@ If the EventListener is exposed outside of the cluster, the user must also sign
 - Release Pipeline: User would like to create releases of their project on a nightly basis.
 
 ## Proposal
+Create ScheduledRun CRD with `resourcetemplates` and `schedule` field
 
+This solution doesn't use TriggerTemplate or TriggerBindings. Instead, it's spec has `resourcetemplates` which can have one or more pipelineruns, taskruns or other allowed Kubernetes and Tekton resources.
+
+```yaml
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: ScheduledRun
+metadata:
+  name: nightly-release
+spec:
+  cloudEventSink: http://eventlistener.free.beeceptor.com
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      generateName: simple-pipeline-run-
+    spec:
+      pipelineRef:
+        name: simple-pipeline
+      params:
+      - name: message
+        value: "Hi"
+      - name: git-url
+        value: "https://github.com/tektoncd/community.git"
+      workspaces:
+      - name: git-source
+        emptyDir: {}
+    schedule: "0 0 * * *"
+```
+
+`ScheduledRun` reconciler which creates resources can be part of Triggers but we can move it to Pipelines also.
+
+### CLI
+
+As part of future work for this proposal, we can add a CLI command to manually trigger execution of a ScheduledTrigger
+similar to the kubectl functionality `kubectl create job --from=cronjob/my-cronjob`, for example:
+
+```sh
+tkn crontrigger start <scheduled_trigger_cr_name>
+```
+
+This functionality is tracked in https://github.com/tektoncd/cli/issues/1833.
+
+## Design Evaluation
+
+### Reusability
+
+This feature is essentially syntactic sugar for a CronJob creating a Tekton resource.
+It uses functionality similar to EventListeners, such as TriggerGroups and CloudEventSinks (but does not actually create an EventListener).
```

Comment:
This may need to be updated if we're not using TriggerGroups

Reply:
I changed this to the below line:
```
It uses the functionality of TriggerTemplate but does not depend upon an EventListener.
```